### PR TITLE
Create exercise from training

### PR DESCRIPTION
**Pull request by PerAsperaAdAstra3**
Exercise selection for ExerciseInRound improved, selection in exercise field now enabled trough drop-down combo that is filtered as letters are typed in to input field. Adding new Exercise directly from training page has been implemented.

### CODE REVIEW

File: src/main/resources/static/scripts/ajaxFunctions.js [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+	
+	exerciseInRound["exerciseInRoundExerciseName"] = $("#exerciseInRoundExerciseName").val();
+	exerciseInRound["exerciseInRoundExerciseId"] = $("#exerciseInRoundExerciseId").val();
+	exerciseInRound["idExerciseInRound"] = $("#idExerciseInRound").val();
+	exerciseInRound["exerciseInRoundNote"] = $("#exerciseInRoundNote").val();
+	exerciseInRound["exerciseInRoundNumberOfRepetitions"] = $("#exerciseInRoundNumberOfRepetitions").val();
+	exerciseInRound["exerciseInRoundDifficulty"] = $("#exerciseInRoundDifficulty").val();
+	exerciseInRound["roundId"] = $("#hiddenRoundInTraining").val();
+	
+$.ajax({
```
Comment (renatav) [inline src/main/resources/static/scripts/ajaxFunctions.js]: Don't know if it's just GitHub, but indentation looks weird here

File: src/main/resources/templates/trainingCreation.html [hunk 2]
```diff
@@ -71,11 +71,30 @@
 				areFieldsFilledIn = false;
 				$("#errorExerciseId").show();
 			}
-
-			if(areFieldsFilledIn){	
-				 ajaxExerciseInRound();
+			var isNotSame = checkExerciseNameTest()
+			
+			if(isNotSame == true){
```
Comment (renatav) [inline src/main/resources/templates/trainingCreation.html]: `if (isNotSame)`

File: src/main/resources/templates/trainingCreation.html [hunk 3]
```diff
@@ -232,8 +253,26 @@
     document.getElementById("myDropdown").classList.toggle("show");
 }
 
+function checkExerciseNameTest() {
+		var exerciseNames = $(".dinamicSelectExerciseName");
```
Comment (renatav) [inline src/main/resources/templates/trainingCreation.html]: Indentation

File: src/main/resources/static/scripts/ajaxFunctions.js (hunk 1 above)
Comment (PerAsperaAdAstra3) [reply src/main/resources/static/scripts/ajaxFunctions.js]: Yap, it's just GitHub, in Eclipse this indentation anomaly is not present.

**Pull request merged**